lib/flipper/adapters/active_record.rb

```diff
@@ -7,34 +7,36 @@ module Adapters
     class ActiveRecord
       include ::Flipper::Adapter
 
-      # Abstract base class for internal models
-      class Model < ::ActiveRecord::Base
-        self.abstract_class = true
-      end
+      ActiveSupport.on_load(:active_record) do
```

Comment:
Would be simpler to just move all the things that depends on Active Record into another file and then require it when AR is loaded, or something like that.

Reply:
There's already a generator for adding the DB migrations, so another option is to just have the generator add these models to the `app/models` directory.